setupEnv.sh

```diff
@@ -1,6 +0,0 @@
-#! /bin/bash
-
-set -e
-
-source dcl-env
-cp .ci/.env.${ENVIRONMENT} .env
```

Comment:
We don't have the .env files per environment anymore, but, we still need a .env file to be loaded to load some variables, for example, the `PUBLIC_PATH`. We still need something to build the `.env` file, don't we?

Reply:
This was only used by the old CI, the current one doesn't need this script, it uses the `prebuild.js` one: https://github.com/decentraland/builder/blob/master/scripts/prebuild.js